dashboard/app/models/lesson.rb

```diff
@@ -223,7 +223,7 @@ def summarize(include_bonus_levels = false)
         title: localized_title,
         lesson_group_display_name: lesson_group&.localized_display_name,
         lockable: !!lockable,
-        levels: cached_levels.map {|l| l.summarize(false)},
+        levels: cached_levels.map {|sl| sl.summarize(false, for_edit: for_edit)},
```

Comment:
note: we should no longer have to pass in `for_edit` here after [PLAT-460](https://codedotorg.atlassian.net/browse/PLAT-460) is done.